charts/node/values.yaml

```diff
@@ -1,272 +1,333 @@
-## ref: https://docs.substrate.io/tutorials/v3/private-network/
+# -- Deploy a substrate node.
+# ref: https://docs.substrate.io/tutorials/v3/private-network/
 node:
+  # -- Name of the chain
   chain: "polkadot"
+  # -- Command to run within the container
   command: "polkadot"
+  # -- Number of replicas to deploy
   replicas: 1
+  # -- Type of the node. One of: full, authority, validator, collator, light
   role: full
+  # -- How node updates should be applied.
   updateStrategy:
-    ## How node updates should be applied
-    ## Type supports RollingUpdate or OnDelete
-    ## maxUnavailable can be an int or a %
-    ##
-    type: "RollingUpdate"
+    # -- Enable custom updateStrategy
     enabled: false
+    # -- Type supports RollingUpdate or OnDelete
+    type: "RollingUpdate"
+    # -- Can be an int or a %
     maxUnavailable: 1
 
 
-  ## Node may require custom name for chainspec file.
-  ## ref:  moonbeam https://github.com/PureStake/moonbeam/issues/1104#issuecomment-996787548
-  ## Note: path should start with /chain-data/ since this folder mount in init container download-chainspec.
-  ##
+  # -- Node may require custom name for chainspec file.
+  # ref:  moonbeam https://github.com/PureStake/moonbeam/issues/1104#issuecomment-996787548
+  # Note: path should start with /chain-data/ since this folder mount in init container download-chainspec.
   customChainspecPath: "/chain-data/chainspec.json"
-  # customChainspecUrl:
+  # -- (string) URL to retrive custom chain spec
+  customChainspecUrl:
+  # -- Replace chain spec if it already exists
   forceDownloadChainspec: false
 
   chainData:
-    ## Database backend engine to use
-    ##
+    # -- Database backend engine to use
     database: rocksdb
-    ## Set the amount of blocks to retain
-    ##
-    # pruning: 1000
-    ## Configuration to download chain snapshot from remote URL
-    ##
-    # snapshotUrl: "https://dot-rocksdb.polkashots.io/snapshot"
-    # snapshotFormat: lz4
-    # kubernetesVolumeSnapshot: ""
-    # kubernetesVolumeToClone: ""
-    # GCSBucketUrl: ""
-    # chainPath: ""
-    ## Storage class to use for persistent volume
-    ##
+    # -- Set the amount of blocks to retain. If set to 0 archive node will be run
+    pruning: 1000
+    # -- (string) If set, create a clone of the volume (using volumeClaimTemplates.dataSource.VolumeSnapshot) and use it to store chain data
+    kubernetesVolumeSnapshot:
+    # -- (string) If set, create a clone of the volume (using volumeClaimTemplates.dataSource.PersistentVolumeClaim) and use it to store chain data
+    kubernetesVolumeToClone:
+    # -- (string) Path on the volume to store chain data
+    chainPath:
+    # -- Storage class to use for persistent volume
     storageClass: ""
+    # -- Size of the volume for chain data
     volumeSize: 100Gi
+    # -- Annotations to add to the volumeClaimTemplates
     annotations: {}
 
+    # -- Configure parameters for restoring chain snapshot. Uses [rclone](https://rclone.org/)
```

Comment:
Truly excellent design of the snapshot configuration. Could you also add a variable for passing custom flags to rclone in case this is needed ?

Reply:
Never mind I see this is already possible in the `downloadChainSnapshot` block.

Reply:
it is there: `initContainers.downloadChainSnapshot.cmdArgs`